nixos/modules/programs/foot/default.nix

```
@@ -0,0 +1,79 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.foot;
+
+  settingsFormat = pkgs.formats.iniWithGlobalSection { listsAsDuplicateKeys = true; };
+in
+{
+  options.programs.foot = {
+    enable = lib.mkEnableOption "foot terminal emulator";
+
+    package = lib.mkPackageOption pkgs "foot" { };
+
+    settings = lib.mkOption {
+      inherit (settingsFormat) type;
+      default = {
+        globalSection = { };
+      };
+      description = ''
+        Configuration for foot terminal emulator. Further information can be found in foot's man page foot.ini(5).
+      '';
+    };
+
+    theme = lib.mkOption {
+      type = with lib.types; nullOr str;
+      default = null;
+      description = ''
+        Include theme from https://codeberg.org/dnkl/foot/src/branch/master/themes.
```

- make the link clickable
- `include` is an implementation detail so it should not be in the description.

```suggestion
        Theme name.  Check <https://codeberg.org/dnkl/foot/src/branch/master/themes> for available themes.
```